src/main/java/com/amazon/janusgraph/example/MarvelGraphFactory.java

```diff
@@ -65,9 +65,9 @@
     public static final String WEAPON = "weapon";
     public static final MetricRegistry REGISTRY = MetricManager.INSTANCE.getRegistry();
     public static final ConsoleReporter REPORTER = ConsoleReporter.forRegistry(REGISTRY).build();
-    private static final String TIMER_LINE = "MarvelGraphFactory.line";
```

Comment:
These were changes to revert MarvelGraphFactory.java back to its original state.